utils/system.py

```diff
@@ -6,22 +6,24 @@
 def parse_params():
     parser = argparse.ArgumentParser(description='FakeNewsChallenge fnc-1-baseline')
     parser.add_argument('-c', '--clean-cache', action='store_true', default=False, help="clean cache files")
+    parser.add_argument('-r',  action="store_false", default=True, dest="is_full_set",help='related or not')
     params = parser.parse_args()
 
     if not params.clean_cache:
-        return
+        pass
+    else:
+        dr = "features"
+        for f in os.listdir(dr):
+            if re.search('\.npy$', f):
+                fname = os.path.join(dr, f)
+                os.remove(fname)
+        for f in ['hold_out_ids.txt', 'training_ids.txt']:
+            fname = os.path.join('splits', f)
+            if os.path.isfile(fname):
+                os.remove(fname)
+        print("All clear")
 
-    dr = "features"
-    for f in os.listdir(dr):
-        if re.search('\.npy$', f):
-            fname = os.path.join(dr, f)
-            os.remove(fname)
-    for f in ['hold_out_ids.txt', 'training_ids.txt']:
-        fname = os.path.join('splits', f)
-        if os.path.isfile(fname):
-            os.remove(fname)
-
-    print("All clear")
+    return True if params.is_full_set else False
```

Comment:
We shouldn't just return true or false. What happens if we need to add a new option in the future?

Reply:
We can use enum to make it more clear: https://docs.python.org/3/library/enum.html